### PR TITLE
39 alter sensor table and add 2 endpoints to fetch data from geostreams

### LINKED ISSUE #39: Add weather sensor field
Add the weather sensor field in the fields response

### PR DESCRIPTION
**Pull request by Rashmil-1999**
This PR alters the sensor table and adds 2 new endpoints to fetch data from geostreams. With this update, you will need to download the latest data.json file from the box and run the import script again.

### CODE REVIEW

File: .github/workflows/pytest.yml [hunk 1]
```diff
@@ -64,3 +64,4 @@ jobs:
           SERVER_HOST: http://localhost
           POSTGRES_TEST_DB: test_direct4ag
           POSTGRES_DB: direct4ag
+          GEOSTREAMS_API_STR: https://direct4ag.ncsa.illinois.edu/geostreams/api/
```
Comment (navarroc) [inline .github/workflows/pytest.yml]: Does it need the actual value or would a dummy value work here? We could make this a organization secret so we aren't putting the URL in the repo

Comment (Rashmil-1999) [reply .github/workflows/pytest.yml]: I wanted to keep a dummy here but I don't have enough permissions so could you please add a secret and update the file?

Comment (Rashmil-1999) [reply .github/workflows/pytest.yml]: we can do the {{}} thing here...

Comment (navarroc) [reply .github/workflows/pytest.yml]: I'm confused, if I put GEOSTREAMS_API_STR: http://localhost/geostreams/api will the tests work? If so, why not just insert a dummy value. If it needs a real value, then we should use a secret.

File: env-example [hunk 2]
```diff
@@ -5,6 +5,7 @@ POSTGRES_PASSWORD=
 POSTGRES_DB=direct4ag
 POSTGRES_TEST_DB=test_direct4ag
 POSTGRES_PORT=5432
+GEOSTREAMS_API_STR=https://direct4ag.ncsa.illinois.edu/geostreams/api/
```
Comment (navarroc) [inline env-example]: Can you make this http://localhost:9000/api?

**Review by navarroc — APPROVED** [review]

**Pull request merged**